### DISCUSSION TITLE
"revert registry changes" in uninstalling instructions

### DISCUSSION THREAD

**Discussion by bertsons** [pal1000/mesa-dist-win#145] [Q&A] [answered]
I'm uninstalling mesa 3D with these steps https://www.bbc.co.uk/news/world-us-canada-64965277, but I think step 8 is too vague; "Revert any registry changes and any environment variables that configure Vulkan runtime to use any of Mesa3D Vulkan drivers;". I am not sure what the registry changes are or the exact environment variables? I know nothing about Mesa3D or Vulkan.

**Comment by pal1000** [comment, marked as answer]
Made some improvements to docs in 9358541a21720755fbbc6c3735a808826cd0ae1c.  Basically if you never registered any Mesa3D Vulkan and OpenCL driver you should skip that step as there is nothing to revert.
